nni/tools/nni_manager_scripts/collect_gpu_info.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""
+Collect GPU utilization metrics, and debug info if ``--detail`` is specified.
+Results are printed to stdout in JSON format.
+
+See `ts/nni_manager/common/gpu_scheduler/collect_info` for details.
+"""
+
+# pylint: skip-file
```

Comment:
Why skip it?

Reply:
nvml* APIs are not pythonic. Wildcard import, unused import, not snake case, etc.